components/blocks/content.tsx

```diff
@@ -71,8 +76,14 @@ export const contentBlockSchema: Template = {
         customImageBlockSchema,
         clientLogosBlockSchema,
         videoEmbedBlockSchema,
+        colorBlockSchema,
       ],
     },
+    {
+      type: "string",
+      label: "Custom Container Class",
+      name: "containerClass",
+    },
```

Comment:
Can this instead be a boolean or something, like no padding that applies px-0 md:px-8 !py-0?

Reply:
Done - Replaced it with dropdown